cadctap/cadctap/core.py

```diff
@@ -521,41 +637,55 @@ def handle_error(msg, exit_after=True):
     else:
         logging.basicConfig(level=logging.WARN, stream=sys.stdout)
 
-    subject = net.Subject.from_cmd_line_args(args)
+    subject = _get_subject(args)
 
-    client = CadcTapClient(subject, resource_id=args.service)
-    if args.cmd == 'create':
-        client.create_table(args.TABLENAME, args.TABLEDEFINITION,
-                            args.format)
-    elif args.cmd == 'delete':
-        reply = input(
-            'You are about to delete table {} and its content... '
-            'Continue? [yes/no] '.format(args.TABLENAME))
-        while True:
-            if reply == 'yes':
-                client.delete_table(args.TABLENAME)
-                break
-            elif reply == 'no':
-                logger.warn(
-                    'Table {} not deleted.'.
-                    format(args.TABLENAME))
-                sys.exit(-1)
+    try:
+        if ('http:' not in args.service and
+                'https:' not in args.service and
+                'cadc.nrc.ca' not in args.service):
+            if args.service.startswith("/"):
```

Comment:
I don't think we should allow service to start with '/'. Please remove this option.

Reply:
Okay. Removed.